### PR TITLE
ACMS-1599: updated media library dependency for media model

### PR DESCRIPTION
**Pull request by deepakmishra2**
**Motivation**

Fixes ACMS-1599

**Proposed changes**

**Alternatives considered**

**Testing steps**

**Merge requirements**
- [ ] _Major change_, _Minor change_, _Bug_, _Enhancement_, and/or _Chore_ label applied
- [ ] Manual testing by a reviewer

### CODE REVIEW

**Review by apathak18 — APPROVED** [review]
LGTM

File: modules/acquia_cms_common/tests/src/FunctionalJavascript/Ckeditor5ConfigurationTestBase.php [hunk 1]
```diff
@@ -29,6 +29,7 @@ abstract class Ckeditor5ConfigurationTestBase extends WebDriverTestBase {
    */
   protected static $modules = [
     'acquia_cms_common',
+    'acquia_cms_image',
```
Comment (vishalkhode1) [inline modules/acquia_cms_common/tests/src/FunctionalJavascript/Ckeditor5ConfigurationTestBase.php:32]: @deepakmishra2 Why this has been added here ?

Comment (deepakmishra2) [reply modules/acquia_cms_common/tests/src/FunctionalJavascript/Ckeditor5ConfigurationTestBase.php]: @vishalkhode1 added by Ankit, he has removed that now along with media_library module.

**Comment by deepakmishra2** [comment]
Closing this as created a new PR-1386 against develop

**Pull request closed without merging**